resources/profiles/Anycubic/process/0.30mm Draft @Anycubic Kobra2.json

```diff
@@ -0,0 +1,108 @@
+{
+	"type": "process",
+	"setting_id": "GP004",
+	"name": "0.30mm Draft @Anycubic Kobra2",
+	"from": "system",
+	"inherits": "fdm_process_common",
+	"instantiation": "true",
+	"adaptive_layer_height": "1",
+	"reduce_crossing_wall": "0",
+	"layer_height": "0.3",
+	"max_travel_detour_distance": "0",
+	"bottom_surface_pattern": "monotonic",
+	"bottom_shell_layers": "3",
+	"bottom_shell_thickness": "1.2",
+	"bridge_flow": "0.85",
+	"bridge_speed": "40",
+	"brim_width": "3",
+	"brim_object_gap": "0.12",
+	"compatible_printers_condition": "",
+	"print_sequence": "by layer",
+	"default_acceleration": "2500",
+	"top_surface_acceleration": "0",
+	"bridge_no_support": "1",
+	"draft_shield": "disabled",
+	"elefant_foot_compensation": "0.1",
+	"enable_arc_fitting": "0",
+	"outer_wall_line_width": "0.4",
+	"wall_infill_order": "inner wall/outer wall/infill",
+	"line_width": "0.42",
+	"infill_direction": "45",
+	"sparse_infill_density": "10%",
+	"sparse_infill_pattern": "rectilinear",
+	"initial_layer_acceleration": "2000",
+	"travel_acceleration": "0",
+	"inner_wall_acceleration": "0",
+	"initial_layer_line_width": "0.8",
```

Comment:
0.8mm is a bit agressive.
Do you think 0.5 would be better for normal use case(which is the built-in profile built for)?

Reply:
@SoftFever Yes it does sound like a lot doesn't it! This is the comparative value provided by Anycubic in their Kobra 2 Prusaslicer config - `first_layer_extrusion_width = 200%`. So actually that's 0.84.

Happy with whatever you think is best!